elasticsearch/templates/_helpers.tpl

```diff
@@ -11,9 +11,17 @@ Create a default fully qualified app name.
 We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
 */}}
 {{- define "fullname" -}}
```

Comment:
This template is actually not being used at all. So it could be removed instead.